ytypes/unmarshal.go

```diff
@@ -29,6 +29,40 @@ type UnmarshalOpt interface {
 	IsUnmarshalOpt()
 }
 
+// ComplianceErrors contains the compliance errors encountered from an Unmarshal operation.
+type ComplianceErrors struct {
+	// Generic errors for now, until we make a decision on what specific types of errors should
+	// be returned
+	Errors []error
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences
Comments should begin with the name of the thing being described and end in a period: